docs/sources/reference/commandline/cli.md

```diff
@@ -349,6 +350,31 @@ https://linuxcontainers.org/) via the `lxc` execution driver, however, this is
 not where the primary development of new functionality is taking place.
 Add `-e lxc` to the daemon flags to use the `lxc` execution driver.
 
+### Daemon user namespace support
```

Comment:
@moxiegirl I'm not even saying to keep the sentence the same way, I'm saying that there should be some sort of breadcrumb in there to help people get on the track of understanding the limitation. Just leaving it as, "You cannot do what you want to do", is frustrating. Maybe just referring to another part of the documentation is fine?